aleph-client/src/treasury.rs

```diff
@@ -17,23 +17,13 @@ pub fn treasury_account() -> AccountId32 {
 
 /// Returns how many treasury proposals have ever been created.
 pub fn proposals_counter<C: AnyConnection>(connection: &C) -> u32 {
-    connection
-        .as_connection()
-        .get_storage_value("Treasury", "ProposalCount", None)
```

Comment:
Some thought on naming - we switch here to _read_ from _get_ - I think I find the latter (so current) approach more viable since this is following `substrate-api-client` convention, so the Substrate in general - most of `state` RPC calls are _get_ prefixed.

Reply:
yes, I changed it intentionally so that to differentiate between our wrappers and other 'standard' methods in `Connection` objects; if everything would start with `get` it would be quite misleading; however, if you insist I can change it

Reply:
That's fine, both _read_ and _get_ is fine for me.